locale/fr_FR/LC_MESSAGES/clipboard-indicator.po

```diff
@@ -44,11 +44,11 @@ msgstr "Tout effacer ?"
 
 #: extension.js:340
 msgid "Are you sure you want to delete all clipboard items?"
-msgstr "Etes-vous sûr de vouloir effacer tout effacer ?"
+msgstr "Etes-vous sûr de vouloir tout effacer ?"
 
 #: extension.js:341
 msgid "This operation cannot be undone."
-msgstr "Cette action ne peut ëtre annulée."
+msgstr "Cette action ne peut être annulée."
```

Comment:
Should we also add a `pas` here? So `ne peut pas ...`.

Reply:
Hmm, I wanted to keep it short so I omitted `pas` . But both forms are correct.

Reply:
Sounds good. :+1: